src/import/download_objectstore_files.py

```diff
@@ -75,9 +78,10 @@ def get_grondexploitatie_files():
 
             log.debug('AGE: %d %s', delta.days, expected_file)
 
-            if delta.days > 180:
-                log.error('DELEVERY IMPORTED FILES ARE TOO OLD!')
-                raise ValueError
+            if max_age > 0:
```

Comment:
`if a and b` instead of two ifs?

Reply:
PyCharm wilde dat herschrijven als 0 < delta.days < max_axe. Maar dat vond ik onduidelijk . De andere suggestie van PyCharm was om er twee ifs van te maken. Dat maakt de  bedoeling in ieder geval duidelijk